### ISSUE TITLE
"Start Minimized" not working"

### ISSUE THREAD

**Issue by cinematic6436** [canton7/SyncTrayzor#634]
Hi --

I have a shortcut for SyncTrayzor in my Windows 10 startup directory with "Start Minimized" checked under "File" > "Settings," but it doesn't seem to make a difference -- when my computer starts, SyncTrayzor opens, and I have to click the X in the upper right corner to minimize it to the system tray every time. Is anyone else experiencing this?

Thanks,
Brian

**Comment by canton7** [comment]
See #513? The "Start minimised" checkbox affects how synctrayzor is started from the autostart registry key that it creates. If you're starting it from a shortcut, that setting won't do anything. You'll need to edit the shortcut to pass `-minimized`.

**Comment by canton7** [comment]
(You'll notice that "Start minimized" is disabled when "Automatically start on login" is unchecked)

**Comment by cinematic6436** [comment]
Great! Thanks for the quick reply. Adding "-minimized" after the target in the shortcut did the trick. I misread post 513 -- I had "Run: Minimized" selected and thought that was all that had to be done. Sorry if this was a redundant post.

If anyone else has this problem, the program was working properly after I did the above, but the tray icon disappeared altogether until I unchecked "Show tray icon only on close."

Thanks again!
Brian
